### DISCUSSION TITLE
Can we create multiple schedulers in a model or control the order in running different types of agents?

### DISCUSSION THREAD

**Discussion by cylin928** [mesa/mesa#2153] [General]
In Mesa time module https://mesa.readthedocs.io/en/stable/apis/time.html, we have access to different iterative strategies to run agents. However, can we create multiple schedulers in a model or control the order in running different types of agents?
For example, simulate AgentType2 after all AgentType1 are simulated.

Thanks!

**Comment by wang-boyu** [comment]
Would you like to take a look at an experimental feature `AgentSet`:

- https://github.com/projectmesa/mesa/discussions/1912#discussioncomment-7890448
- https://github.com/projectmesa/mesa/pull/1916
